### PR TITLE
Create michael-dera.md

### PR DESCRIPTION
**Pull request by michaeldera**
Adding speaker information.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=1036) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=1036) before we can accept your contribution.<br/><br/>:x: michaeldera [sign now](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=1036)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet-foundation/website?pullRequest=1036) it.</sub>

File: input/community/speakers/michael-dera.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+Title: Michael Dera
+Pronouns: He
```
Comment (rprouse) [inline input/community/speakers/michael-dera.md:3]: Build failure, Preferred gender pronouns should be of the form [subject]/[object] in lowercase (I.e. "they/them") (ValidateSpeakerPronouns)

**Review by rprouse — CHANGES_REQUESTED** [review]
Other than that, looks good.

**Review by Arhell — APPROVED** [review]

**Comment by rprouse** [comment]
Still waiting on the pronoun fix @michaeldera

**Review by rprouse — APPROVED** [review]

**Pull request merged**